root/static/scripts/edit/externalLinks.js

```diff
@@ -1568,86 +1568,94 @@ function isValidURL(url: string) {
 }
 
 const URL_SHORTENERS = [
-  'adf.ly',
-  'album.link',
-  'ampl.ink',
-  'amu.se',
-  'artist.link',
-  'band.link',
-  'biglink.to',
-  'bit.ly',
-  'bitly.com',
-  'backl.ink',
-  'bruit.app',
-  'bstlnk.to',
-  'cli.gs',
-  'deck.ly',
-  'distrokid.com',
-  'ditto.fm',
-  'eventlink.to',
-  'fanlink.to',
-  'ffm.to',
-  'found.ee',
-  'fty.li',
-  'fur.ly',
-  'g.co',
-  'gate.fm',
-  'geni.us',
-  'goo.gl',
-  'hypel.ink',
-  'hyperurl.co',
-  'is.gd',
-  'kl.am',
-  'laburbain.com',
-  'li.sten.to',
-  'linkco.re',
-  'lnkfi.re',
-  'linktr.ee',
-  'listen.lt',
-  'lnk.bio',
-  'lnk.co',
-  'lnk.site',
-  'lnk.to',
-  'lsnto.me',
-  'many.link',
-  'mcaf.ee',
-  'moourl.com',
-  'music.indiefy.net',
-  'musics.link',
-  'mylink.page',
-  'myurls.bio',
-  'odesli.co',
-  'orcd.co',
-  'owl.ly',
-  'page.link',
-  'pandora.app.link',
-  'podlink.to',
-  'pods.link',
-  'push.fm',
-  'rb.gy',
-  'rubyurl.com',
-  'smarturl.it',
-  'snd.click',
-  'song.link',
-  'songwhip.com',
-  'spinnup.link',
-  'spoti.fi',
-  'sptfy.com',
-  'spread.link',
-  'streamerlinks.com',
-  'streamlink.to',
-  'su.pr',
-  't.co',
-  'tiny.cc',
-  'tinyurl.com',
-  'tourlink.to',
-  'trac.co', // Host links can be legitimate; non-root paths are aggregators
-  'u.nu',
-  'unitedmasters.com',
-  'untd.io',
-  'vyd.co',
-  'yep.it',
-].map(host => new RegExp('^https?://([^/]+\\.)?' + host + '/.+', 'i'));
+  /* eslint-disable sort-keys */
+  {host: 'adf.ly', blockHost: false},
+  {host: 'album.link', blockHost: false},
+  {host: 'ampl.ink', blockHost: false},
+  {host: 'amu.se', blockHost: false},
+  {host: 'artist.link', blockHost: false},
+  {host: 'band.link', blockHost: false},
+  {host: 'biglink.to', blockHost: false},
+  {host: 'bio.link', blockHost: true},
```

Comment:
It might be better to just add the hosts that are blocked to a Set, and check membership of the set, since the added property doesn't vary much.  (I guess most should probably be true, and the set can be inverted at that point.)

Reply:
Is that significantly faster? (this seems clearer to me, but if it's slower then we could do it the other way).

Reply:
It's not faster, but the code can minify better.  How many of these do you estimate should actually be `blockHost: false`?  I would guess only a few...

Reply:
Probably only a few, yes, it's just the annoyance of looking through them all. But maybe I should just do that, and then it's done for the future...

Reply:
Looked through them in b126ea4ee4ca8cff092b65f48446fd92f516c43b